### PR TITLE
GraphCard created, callback removed in Leaderboard, adminDashboard polished.

### PR DESCRIPTION
**Pull request by ashtilawat23**
Description: 
I created a graphcard component to use to display graphs using plotly.js in squares that are the same size. I removed a callback function that was just doing a console.log. I tested the graph card on the adminDash and it worked.

### CODE REVIEW

**Review by eet349 — APPROVED** [review]
Thanks for making those changes. Looks good.

File: src/components/common/GraphCard.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+export default function GraphCard() {
+  return (
+    <div>
+      <Card style={{ width: 650, height: 300 }}>
+        <Plot
+          data={[
+            {
+              x: [-2, -1, 0, 1, 2],
+              y: [1, 2, 4, 3, 2],
+              type: 'scatter',
```
Comment (L-Steinmacher) [inline src/components/common/GraphCard.js:14]: Code looks good.  Clean!

Comment (ashtilawat23) [reply src/components/common/GraphCard.js:14]: Gracias!

File: src/components/pages/Leaderboard/RenderLeaderboard.js [hunk 2]
```diff
@@ -70,7 +67,7 @@ function RenderLeaderboard(props) {
             </li>
           </div>
         ))}
```
Comment (L-Steinmacher) [inline src/components/pages/Leaderboard/RenderLeaderboard.js:69]: Taking out console.logs!  Great Eye!

**Review by L-Steinmacher — APPROVED** [review]
Looks good. Straight forward and clean code.

**Pull request merged**